pdns/dnsdistdist/dnsdist-backend.cc

```diff
@@ -588,7 +588,7 @@ void DownstreamState::updateNextLazyHealthCheck(LazyHealthCheckStats& stats)
       const uint16_t failedTests = currentCheckFailures;
       size_t backOffCoeff = std::pow(2U, failedTests);
       time_t backOff = d_config.d_lazyHealthCheckMaxBackOff;
-      if ((std::numeric_limits<time_t>::max() / d_config.d_lazyHealthCheckFailedInterval) >= backOffCoeff) {
+      if ((static_cast<size_t>(std::numeric_limits<time_t>::max()) / d_config.d_lazyHealthCheckFailedInterval) >= backOffCoeff) {
```

Comment:
I think this does not work on platforms with 32-bit `size_t` and 64-bit `time_t` (e.g all OpenBSD 32-bit platforms). Maybe make `backOffCoeff` a `time_t` instead?
```
time_t backOffCoeff = std::pow(2, failedTests);
```

Reply:
I pushed a new version which should be better, or at least I hope so :)